src/Platforms/MariaDb1052Platform.php

```diff
@@ -6,7 +6,7 @@
 use Doctrine\DBAL\Schema\TableDiff;
 
 /**
- * Provides the behavior, features and SQL dialect of the MariaDB 10.5 (10.5.2 GA) database platform.
+ * Provides the behavior, features and SQL dialect of the MariaDB 10.5 (10.5.2 Beta) database platform.
```

Comment:
10.5.2 is not a GA, but a beta release. Not sure if that suffix provides any value though. Maybe we should drop the suffix (and even the patch level) from this documentation, since nobody can really expect us to support ancient alpha/beta releases of any database.

Reply:
Specifying the patch level allows us to explicitly say that we don't support 10.5.1, and never will, so  there's that. I agree that it's not really our role to remind our users what stability a specific patch version is.

Reply:
I kept the patch level in the docs now, but dropped them from the platform classes. WDYT?